docs/faq.md

```diff
@@ -17,31 +17,55 @@ And with the introduction of the jump to date API via
 [MSC3030](https://github.com/matrix-org/matrix-spec-proposals/pull/3030), we could show
 messages from any given date and day-by-day navigation.
 
-## How do I opt out and keep my room from being indexed by search engines?
-
-All public Matrix rooms are accessible to view in the Matrix Public Archive. But only
-rooms with history visibility set to `world_readable` are indexable by search engines.
-
-Also see https://github.com/matrix-org/matrix-public-archive/issues/47 to track better
-opt out controls.
+## Why did the archive bot join my room?
+
+Only public Matrix rooms with `shared` or `world_readable` [history
+visibility](https://spec.matrix.org/v1.7/client-server-api/#room-history-visibility) are
```

Comment:
Is the spec expected to greatly change or break the link or will this be updated every time a new spec releases?

```suggestion
visibility](https://spec.matrix.org/latest/client-server-api/#room-history-visibility) are
```

Reply:
It doesn't really matter in this case because `v1.6` and `v1.7` have the information we're trying to explain.

Using `latest` means we get any further clarifications added but also means that the link is probably going to break at some point.

I'm fine with using `latest` for now though :fast_forward: